Resources/doc/index.rst

```diff
@@ -41,7 +41,7 @@ following:
     {
         $bundles = array(
             //...
-            new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
+            $bundles[] = new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle();
```

Comment:
sorry... I was trying to standardize bundle installation, but code was for a dev/test bundle. Sorry.